releasenotes-builder/src/test/java/com/github/checkstyle/templates/TemplateProcessorTest.java

```diff
@@ -252,6 +252,36 @@ public void testGenerateCustomTemplate() throws Exception {
         assertFile("customTemplate.txt", MainProcess.GITHUB_FILENAME);
     }
 
+    @Test
+    public void testVersionValidationForBreakingCompatibility() throws Exception {
+        final List<String> errors = MainProcess.runPostGenerationAndPublication(
+            createNotes(createAllNotes(), null, null, null, null), createBaseCliOptions()
+                .setGenerateAll(true).build());
+
+        Assert.assertEquals("no version validation error", 0, errors.size());
```

Comment:
set setValidateVersion(true), 

remove unrelated asserts.
you need to assert only fact that version validation is with or without errors.

Reply:
Changes Done